src/main.rs

```diff
@@ -20,23 +20,42 @@ async fn main() -> Result<()> {
     let app_matches = App::new(grpc::AGENT_NAME)
         .version("1.0.0")
         .arg(
-            Arg::with_name("socket addr")
-                .long("grpc_sock")
+            Arg::with_name("KeyProvider gRPC socket addr")
+                .long("keyprovider_sock")
                 .takes_value(true)
-                .help("The socket address which the grpc service will listen to, for example: --grpc_sock 127.0.0.1:11223",
+                .help("This socket address which the KeyProvider gRPC service will listen to, for example: --keyprovider_sock 127.0.0.1:11223",
+                ),
+        )
+        .arg(
+            Arg::with_name("GetResource gRPC socket addr")
```

Comment:
Why create two different sockets for this? In practice won't `ocicrypt-rs` be the one using both? I guess it might be more robust to have a separate socket?

Reply:
My considerations of using two separate sockets are as follows:

1. In fact, the caller of get resource is not ocicrypt RS, but image rs. ocicrypt RS does not support container image signature verification, but only image encryption and decryption. Image RS actually completes image signature verification. (this follows the implementation logic of ocicrypt and image in golang language version)

2. The protobuf of the key provider grpc service has been defined by ocicrypt. In order to maintain compatibility as much as possible, we'd better not change it.